requirements_dev.txt

```diff
@@ -2,7 +2,7 @@ pytest
 pytest-subtests
 pytest-cov
 mpi4py
-git+https://github.com/spcl/dace.git@v0.14rc2
+dace>=0.14
```

Comment:
Still unsure about this requirement here, since it's in all the other relevant packages. I freshly cloned pace, removed this line, and was still able to `make constraints.txt` with the same `dace==0.14` without problem.

Reply:
Yes this is no longer needed since this is not a tag, I'll remove, good catch

Reply:
It was only needed because of the "git version can't be in constraint.txt" new behavior of PIP